src/main/java/uk/gov/hmcts/darts/cache/token/service/value/impl/ServiceContextCacheValue.java

```diff
@@ -45,9 +47,14 @@ public ServiceContextCacheValue(ServiceContext context) throws CacheException {
         }
     }
 
-    public ServiceContextCacheValue(ServiceContextCacheValue value) {
-        setId(value.getId());
-        setContextString(value.getContextString());
+    public ServiceContextCacheValue(ServiceContextCacheValue value) throws CacheException {
+        try {
+            jaxbContext = JAXBContext.newInstance(ServiceContext.class);
+            setId(value.getId());
```

Comment:
Looking at this again, I'm not really sure why we need these 2 setter methods? Please check.
They also look suspect to me also because they have `throws CacheException` on the method signature